### ISSUE TITLE
Version 9 Typings for auth.ProviderId

### ISSUE THREAD

**Issue by cdcarson** [firebase/firebase-js-sdk#4853]
#### Describe your environment

  * Operating System version: MacOs 11.2.3 
   * Browser version: n/a
  * Firebase SDK version: 9.0.0-beta.1
  * Firebase Product: auth

#### Bug
There seems to be a problem with the typings in @firebase/auth/dist/auth-exp-public.d.ts. On a freshly created Angular app I'm getting the following error trying to import `getAuth`:

```
Error: node_modules/@firebase/auth/dist/auth-exp-public.d.ts:3576:28 - error TS2694: 
Namespace '".../node_modules/@firebase/auth/dist/auth-exp-public".ProviderId' has no exported member 'FIREBASE'.
```

#### Steps to reproduce:
- create an Angular app with `ng new`
- import `getAuth`
#### Relevant Code:

```ts
// app.component.ts
import { Component } from '@angular/core';
import { initializeApp, FirebaseApp } from 'firebase/app';
import { getAuth } from 'firebase/auth';
import { environment } from '../environments/environment';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.scss']
})
export class AppComponent {

  constructor(
  ) {
    const app: FirebaseApp = initializeApp(environment.firebase);
    console.log(app)
  }

}

```
Note that if you comment out the `getAuth` import, the Angular app compiles, and the Firebase app is initialized properly.

**Comment by Feiyang1** [comment]
It has been fixed by https://github.com/firebase/firebase-js-sdk/pull/4821. It will be released in the next release.

**Comment by Feiyang1** [comment]
It has been fixed in `9.0.0-beta.2`.
